docs/use-signalr-service.md

```diff
@@ -0,0 +1,106 @@
+# Use Azure SignalR Service
+
+- [Provision a Service Instance](#provision)
+- [Install and Use Service SDK](#use-sdk)
+- [Configure Connection String](#connection-string)
+- [Configure Service Options](#service-options)
+- [Scale Out Application Server](#scaleout)
+
+## Provision an Azure SignalR Service instance
+
+Go to [Azure Portal](https://portal.azure.com) to provision a SignalR service instance.
+
+## Install and Use Service SDK
+
+Run below command to install SignalR Service SDK to your ASP.NET Core project.
+
+```bash
+dotnet add package Microsoft.Azure.SignalR --version 1.0.*
+```
+
+In your `Startup` class, use SignalR Service SDK as the following code snippet.
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddSignalR()
+            .AddAzureSignalR();
+}
+
+public void Configure(IApplicationBuilder app)
+{
+    app.UseAzureSignalR(routes =>
+    {
+        routes.MapHub<YourHubClass>("/path_for_your_hub");
+    });
+}
+```
+
+## Configure Connection String
+
+There are two approaches to configure SignalR Service's connection string in your application.
+- Set an environment variable with name `Azure:SignalR:ConnectionString` or `Azure__SignalR__ConnectionString`.
+    - In Azure App Service, put it in application settings.
+- Pass the connection string as a parameter of `AddAzureSignalR()` as below sample codes.
+
+    ```csharp
+    services.AddSignalR()
+            .AddAzureSignalR("Endpoint=...;AccessKey=...;");
```

Comment:
Lets not show any documentation directly passing the key 😄 . People will copy and paste. We can show reading a different configuration key or something else. *ANYTHING* but putting it there directly 😄

Reply:
Can we make it not compile?